test/todo_test.rb

```diff
@@ -23,6 +23,32 @@ def test_add_adds_item
     assert_equal(expected, results)
   end
 
+  def test_item_already_added_is_true
+    test_file = StringIO.new
+    test_file.write("First", "\n")
+    config = {
+      database: test_file
+    }
+    test_manager = Todo::Manager.new(config)
+    item = "First"
+    test_file.rewind
+    results = test_manager.item_already_added?(item)
+    assert_equal(true, results)
```

Comment:
As with the other test, we should use `assert` here.
```suggestion
    assert(results)
```

Reply:
Changing, thanks!